packages/repository/src/repositories/legacy-juggler-bridge.ts

```diff
@@ -37,9 +34,9 @@ export const ModelBaseConstructor = jugglerModule.ModelBaseClass as typeof juggl
  * @param ds {DataSource} Data source
  * @returns {} The new model class with DAO (CRUD) operations
  */
-export function bindModel<T extends typeof juggler.ModelBase>(
+export function bindModel<T extends legacy.ModelBaseClass>(
   modelClass: T,
-  ds: juggler.DataSource,
+  ds: legacy.DataSource,
```

Comment:
Is it possible to use `juggler.DataSource` here -- the namespace being exported above?